project_task_recurring_activity/models/recurring_activity.py

```
@@ -0,0 +1,118 @@
+        for activity in recurring_activities:
+            activity.project_task_id.activity_schedule(
+                activity_type_id=activity.activity_type_id.id,
+                user_id=activity.user_id.id,
+                note=activity.description,
+                summary=activity.summary,
+                date_deadline=fields.Date.today(),
+            )
+            activity.write({"next_recurrence_date": activity._get_next_date()})
+
+    @api.constrains("user_id")
+    def _check_user_id(self):
```

Ruff (0.11.9)

Missing return type annotation for private function `_check_user_id`

Add return type annotation: `None`

(ANN202)